### PR TITLE
Update UniversalDIT_Defaultstyle.json

### PR DESCRIPTION
**Pull request by IceLancerSR**
evdev dualshock L2 R2 correction

### CODE REVIEW

**Comment by IceLancerSR** [comment]
All files needs to be changed to this
Instead TL2 and TR2 should be
Full Axis 2+ and Full Axis 5+

**Comment by Venomalia** [comment]
you would leave the TL2 and TR2.
If you would also change the other files in #Assets, I would accept the pull and your changes would be applied to all other files.

you can make the changes all in this pull request that is clearer, therefore I close the second.

File: #Assets/UniversalDIT_Defaultstyle.json [hunk 1]
```diff
@@ -722,8 +722,8 @@
       "NORTH": "../#DefaultDevices/DualShock4/Defaultstyle/Triangle.png",
       "TL": "../#DefaultDevices/DualShock4/Defaultstyle/L1.png",
       "TR": "../#DefaultDevices/DualShock4/Defaultstyle/R1.png",
-      "`TL2`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
-      "`TR2`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png",
+      "`Full Axis 2+`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
+      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png",
```
Comment (Venomalia) [inline #Assets/UniversalDIT_Defaultstyle.json:726]:
```
      "`TL2`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
      "`TR2`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png",
      "`Full Axis 2+`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png",
```

**Comment by IceLancerSR** [comment]
Sure, done

**Pull request closed without merging**